docs/meshcloud.project-metering.md

```diff
@@ -84,8 +94,21 @@ period. This preview is periodically updated and contains only reports available
 
 A chart under **Financials** > **Chargeback Statements** in the project control plane shows the total amount charged as of the last chargeback statements to get a quick overview of your project cost.
 
+Chargeback statements consist of **line items** showing the individual booking entries made into a chargeback account.
+When booking a tenant usage report, meshStack splits up the report's `netAmount` by seller and product group into
```

Comment:
When booking a tenant usage report, meshStack splits up the report's `netAmount` by seller and product group, into...